### ISSUE TITLE
Installation of Qt5/5.15.2  on top of EESSI 2023.04 compat layer

### ISSUE THREAD

**Issue by TopRichard** [EESSI/software-layer#253]
The installation of Qt5/5.15.2 is failing with EasyBuild v4.7.1 on top of the EESSI pilot 2023.04 compat layer, with:
```
note:   template argument deduction/substitution failed:
../../3rdparty/chromium/third_party/breakpad/breakpad/src/client/linux/handler/exception_handler.cc:141:65: note:   mismatched types std::initializer_list<_Tp> and int
  141 |   static const unsigned kSigStackSize = std::max(16384, SIGSTKSZ);
      |                                                                 ^
[3395/22483] CXX obj/third_party/breakpad/client/linux_ptrace_dumper.o
[3396/22483] CXX obj/third_party/breakpad/client/linux_core_dumper.o
[3397/22483] CXX obj/third_party/breakpad/client/microdump_writer.o
[3398/22483] CXX obj/third_party/breakpad/client/linux_dumper.o
[3399/22483] CXX obj/third_party/breakpad/client/minidump_writer.o
[3400/22483] ACTION //third_party/blink/renderer/core:make_core_generated_css_value_keywords(/tmp/nessibot/easybuild/build/Qt5/5.15.2/GCCcore-10.3.0/qt-everywhere-src-5.15.2/qtwebengine/src/toolchain:target)
[3401/22483] ACTION //third_party/blink/renderer/bindings:web_idl_database(/tmp/nessibot/easybuild/build/Qt5/5.15.2/GCCcore-10.3.0/qt-everywhere-src-5.15.2/qtwebengine/src/toolchain:target)
ninja: build stopped: subcommand failed.
make[4]: *** [Makefile.gn_run:1175: run_ninja] Error 1
```
check: https://github.com/meta-qt5/meta-qt5/blob/2a38fca150f065f869ed530fffe1a07beec80692/recipes-qt/qt5/qtwebengine/chromium/0013-chromium-breakpad-fix-build-with-glibc-2.34.patch

**Comment by boegel** [comment]
We actually have this patch in place already, but only for sufficiently recent `Qt5` easyconfigs, not for `Qt5-5.15.2-GCCcore-10.3.0.eb`, see https://github.com/easybuilders/easybuild-easyconfigs/pull/17165.

I've opened https://github.com/easybuilders/easybuild-easyconfigs/pull/18087 to add the patches also to `Qt5-5.15.2-GCCcore-10.3.0.eb`.

@TopRichard Can you check with `eb --from-pr 18087` whether that fixes the problem you were seeing?

**Comment by boegel** [comment]
@TopRichard This can be closed since we have Qt5 v5.15.2 in place in EESSI 2023.06 (cfr. #284)
